azure/src/services/azureFunctionCloudService.test.ts

```diff
@@ -5,11 +5,11 @@ import {
   CloudService,
   StringParams
 } from "@multicloud/sls-core";
-import { AzureFunctionCloudService, AzureCloudServiceOptions } from ".";
+import { AzureFunctionCloudService, AzureCloudServiceOptions, buildURL } from ".";
 
 const getCartAzureCloudService: AzureCloudServiceOptions = {
   name: "azure-getCart",
-  http: "test-url",
+  http: "test-url/",
```

Comment:
would you consider to remove all the http, methods and headers use them in a variable and call that instead of repeat in every test the same lines?

Reply:
There are a few tests that are not using the same, and have some changes. I will leave a commit, if see it if you cant to revert this changes let me know

Reply:
Thanks for this feedback, it will be applied in my next commit.